testing/internal/e2e/scopes/scope_test.go

```diff
@@ -0,0 +1,49 @@
+package scope_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/boundary/testing/internal/e2e/boundary"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestListKeyVersionDestructionJobsCli uses the boundary CLI to list
+// pending key version destruction jobs.
+func TestListKeyVersionDestructionJobsCli(t *testing.T) {
+	e2e.MaybeSkipTest(t)
+
+	output := boundary.AuthenticateCli()
+	require.NoError(t, output.Err, string(output.Stderr))
+
+	output = e2e.RunCommand(
+		"boundary", "scopes", "list-key-version-destruction-jobs",
+		"-scope-id", "global",
+		"-format", "json",
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+	var keyVersionDestructionJobListResult scopes.KeyVersionDestructionJobListResult
+	err := json.Unmarshal(output.Stdout, &keyVersionDestructionJobListResult)
+	require.NoError(t, err)
+	assert.Empty(t, keyVersionDestructionJobListResult.Items)
+}
+
+// TestListKeyVersionDestructionJobsApi uses the boundary go api to list
+// pending key version destruction jobs.
+func TestListKeyVersionDestructionJobsApi(t *testing.T) {
+	e2e.MaybeSkipTest(t)
+
+	client, err := boundary.NewApiClient()
+	require.NoError(t, err)
+
+	ctx := context.Background()
+
+	scopeClient := scopes.NewClient(client)
+	keyVersionDestructionJobListResult, err := scopeClient.ListKeyVersionDestructionJobs(ctx, "global")
+	require.NoError(t, err)
+	assert.Empty(t, keyVersionDestructionJobListResult.Items)
+}
```

Comment:
CC @moduli let me know if its premature to add tests to this framework.

Reply:
Nope! I think it should be ok. Two things...
1. We'll need to add an enos scenario for this. Based on what I see here, it looks like it will just be a duplicate of `enos-scenario-e2e-target.hcl`, but you can get rid of the `create_target` step and update `test_package` to point to this
2. There's a slight change to `RunCommand` coming in https://github.com/hashicorp/boundary/pull/2474 that will make it look something like this
```
output = e2e.RunCommand("boundary", e2e.WithArgs("scopes", "list-key-version-destruction-jobs",
		"-scope-id", "global",
		"-format", "json",
	))
```

Reply:
Also, maybe nit, but I might suggest changing the directory name from `scopes` to `scope`

Reply:
I removed these for now, will readd once we have key destruction commands.